test/core/services/PlaceholdersSpec.scala

```diff
@@ -77,6 +77,11 @@ class PlaceholdersSpec extends BaseSpec {
     "translate CY:short" in {
       pls.translate("Tax year start: [timescale:CY:short]", today) shouldBe s"Tax year start: 20"
     }
+
+    "translate date literals" in {
+      pls.translate("A date literal: [timescale:23/5/1999]", today) shouldBe s"A date literal: 23/5/1999"
```

Comment:
I think you could dispense with the the argument "today" as the default is "today".

Reply:
could be either, but this usage is consistent with all other tests